### PR TITLE
A11y

### PR DESCRIPTION
**Pull request by peoray**
#111

### CODE REVIEW

File: packages/vue/config/storybook/home/home.js [hunk 1]
```diff
@@ -16,32 +16,32 @@ storiesOf(" |Introduction", module)
     <div class="intro">
       <h1>StorefrontUI - Customization-first component library</h1>
       <p>
-        Welcome in our storybook!
+        Welcome to our storybook!
```
Comment (mayashavin) [inline packages/vue/config/storybook/home/home.js]: this should be in a separate PR

File: packages/vue/config/storybook/home/home.js [hunk 2]
```diff
@@ -16,32 +16,32 @@ storiesOf(" |Introduction", module)
...
         You can use it to browse and play with StorefrontUI components.
-        On the left panel you will find every component that is currently available in the library. 
+        On the left panel, you will find every component that is currently available in the library. 
         You can play with props and CSS modifiers of each component from <b>Knobs menu</b>. 
-        Components marked as <i>WIP</i> will much likely change and they;re not safe to use.
-        On the <b>notes</b> menu you will find additional information about component such as available SCSS variables.
+        Components marked as <i>WIP</i> will much likely change and they're not safe to use.
```
Comment (mayashavin) [inline packages/vue/config/storybook/home/home.js]: `Components marked as <i>WIP</i> will likely be changed, hence it's not safe to use them.`

File: packages/vue/config/storybook/home/home.js [hunk 3]
```diff
@@ -16,32 +16,32 @@ storiesOf(" |Introduction", module)
...
+        On the <b>notes</b> menu, you will find additional information about component such as available SCSS variables.
```
Comment (mayashavin) [inline packages/vue/config/storybook/home/home.js]: Remove this line since we don't use Notes menu anymore. All moved to storybook.

File: packages/vue/config/storybook/home/home.js [hunk 4]
```diff
@@ -16,32 +16,32 @@ storiesOf(" |Introduction", module)
...
       </p>
       <p>
         <b>StorefrontUI is currently in ALPHA, which means it is not suitable for production! Expect things to be broken and APIs to possibly change.</b>
       </p>
       <h2> Join us! </h2>
-        <p> StorefrontUI is a completely new approach for component libraries oriented around customization possibilities instead of out of the box possibilities. It's a great goal but it will take a lot of time to achieve it only by ourselves. The project is open source and free to use. If you'd like to support us and join the team please write to me: <b> filip [at] vuestorefront.io </b>;)</p>
+        <p> StorefrontUI is a completely new approach for component libraries oriented around customization possibilities instead of out of the box possibilities. It's a great goal but it will take a lot of time to achieve it only by ourselves. The project is open source and free to use. If you'd like to support us and join the team, please write to me: <b> filip [at] vuestorefront.io </b>;)</p>
```
Comment (mayashavin) [inline packages/vue/config/storybook/home/home.js]: `StorefrontUI is a completely new approach for component libraries. It is oriented around customisation possibilities instead of out of the box possibilities.`

Comment (mayashavin) [reply packages/vue/config/storybook/home/home.js]: `It's a great goal, but it will take a lot of time to achieve it only by ourselves.`

File: packages/vue/config/storybook/home/home.js [hunk 5]
```diff
@@ -16,32 +16,32 @@ storiesOf(" |Introduction", module)
...
       <h2> How to start? </h2>
       <ol>
         <li> Add the package: <pre>npm install --save @storefrontui/vue</pre> </li>
-        <li> Import stylesheets in App.vue or main.js <pre>import "@storefrontui/vue/dist/css/all.scss";</pre></li>
-        <li> It's done! Now you can import and use any of the components <pre> import SfComponentName from "@storefrontui/vue/dist/SfComponentName.vue"</pre></li>
-        <li> You can find detailed information about every component inside each components stories </li>
+        <li> Import stylesheets in App.vue or main.js: <pre>import "@storefrontui/vue/dist/css/all.scss";</pre></li>
```
Comment (mayashavin) [inline packages/vue/config/storybook/home/home.js]: `<li> Import stylesheets in App.vue or main.js: 
<pre>import "@storefrontui/vue/dist/css/all.scss";</pre>
</li>`

File: packages/vue/config/storybook/home/home.js [hunk 6]
```diff
@@ -16,32 +16,32 @@ storiesOf(" |Introduction", module)
...
+        <li> It's done! Now you can import and use any of the components: <pre> import { SfComponentName } from "@storefrontui/vue/dist/SfComponentName.vue"</pre></li>
```
Comment (mayashavin) [inline packages/vue/config/storybook/home/home.js]: Same here, please split it into lines for readability

File: packages/vue/config/storybook/home/home.js [hunk 7]
```diff
@@ -16,32 +16,32 @@ storiesOf(" |Introduction", module)
...
+        <li> You can find detailed information about every component inside each component stories. </li>
```
Comment (mayashavin) [inline packages/vue/config/storybook/home/home.js]: each component's story

File: yarn.lock [hunk 8]
```diff
@@ -2267,9 +2267,6 @@
     error-stack-parser "^2.0.0"
     string-width "^2.0.0"
 
-"@storefrontui/shared@file:packages/shared":
```
Comment (mayashavin) [inline yarn.lock]: did you run `yarn install` again? Seem like something is missing here.

**Comment by peoray** [comment]
I don't know how to go about separating the commits for each branch. If I create a branch and make a commit, then create another branch and make a commit, the previous commit from the previous commit also shows up.
